src/Webapi/Webapi__Dom/Webapi__Dom_DomStringMap.re

```diff
@@ -0,0 +1,12 @@
+type t = Dom.domStringMap;
+
+type key = string;
+
+[@bs.get_index] [@bs.return nullable]
+external get: (t, key) => option(string) = "";
+[@bs.set_index] external set: (t, key, string) => unit = "";
+let unsafeDeleteKey: (. t, key) => bool = [%raw
+  (map, key) => {|
+            return delete map[key]
+        |}
+];
```

Comment:
I don't think this really needs to return the `bool` value since it'll always be `true` in strict mode. Could you also try to format this on one line if it'll fit, or alternatively two with the entire `[%raw ...]` block on the second line?

I also think it should use t-last, despite being inconsistent with `get_index` and `set_index` it'll be consistent with every other function in `bs-webapi`. You could also obtain consistency internally by providing overrides for `get` and `set` that flip the argument order.

Reply:
Thanks I didn't know `delete` behavior in strict mode, I guess I should learn JS more ^^
I wrapped `get` and `set` too, the inconsistency was too weird.
The `[% raw ...]` formatting was due to refmt that imo adds too many new lines indeed.